HIP/hip-226.md

```diff
@@ -124,7 +130,7 @@ These additional API endpoints do not alter exsiting REST API functionality.
 - Hedera Mirror Node design document
 - Description and code examples of queries added to Hedera REST API documentation section
 - The OpenAPI spec at `api/v1/docs` should be updated to reflect the new endpoints and allow users to test out the calls.
-- Reviewed in Engineeing Insights
+- Reviewed in Engineering Insights
 
 
```

Comment:
We should add what query parameters are supported

Reply:
Added